client/src/components/restaurant/restaurant-item.js

```diff
@@ -36,14 +37,25 @@ class Restaurant extends Component {
         this.setState({
           restaurant: response.data,
           isFetching: true,
+          // eslint-disable-next-line eqeqeq
```

Comment:
Nice! Very interesting message)
You should use `===` operator and explicit type conversion (e.g. `+localStorage.getItem('userID')`